src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -170,6 +170,8 @@ private static String injectRealAccounts(String jsonString) {
                 .replace("${test.student1}", TestProperties.inst().TEST_STUDENT1_ACCOUNT)
                 .replace("${test.student2}", TestProperties.inst().TEST_STUDENT2_ACCOUNT)
                 .replace("${test.instructor}", TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT)
+                .replace("${test.unreg}", TestProperties.inst().TEST_UNREG_ACCOUNT)
+                .replace("${test.admin}", TestProperties.inst().TEST_ADMIN_ACCOUNT)
```

Comment:
Wasn't there a `test.truncated` or something like that? Another question is, how did the test pass even without this? My guess is this code is redundant and should be removed altogether.